packages/LUIS/test/The LUIS cli tool/with 'add' verb.json

```diff
@@ -0,0 +1,9 @@
+[
```

Comment:
I'd suggest moving the test folder hierarchy to json. e.g.
```
{
  "suite" : {
    "title" : "The LUIS cli tool",
    "test"  {
        "prefix" : "with 'add' verb",
      [
        {
          "title" : "and 'application' command",
          "args" : ...
        }
      ]
  }
}
```

Reply:
I would not suggest grouping all the tests for a suite within a single json file.
For `luis.cli.test.suite.js` alone we are talking about over 126, counting only just a single branching for each command line.
The number of tests could easily triplicate o quadruplicate depending how thorough one wants the coverage.